master/docs/manual/mydashboard.html

```diff
@@ -1 +0,0 @@
-../../../smokes/templates/mydashboard.html
```

Comment:
does windows not support symlinks?
Please do not remove them!
We need those symlinks so that we can test the mydashboard in the smoke tests